dienst2/settings.py

```diff
@@ -37,6 +37,10 @@
 # If you set this to False, Django will not use timezone-aware datetimes.
 USE_TZ = True
 
+LOCALE_PATHS = (
+    os.path.join(os.path.dirname(__file__), 'locale'),
```

Comment:
When I remove the line and run `manage.py makemessages` I get `CommandError: Unable to find a locale path to store translations for file manage.py`.
I have no clue of why it would want to translate manage.py